### ISSUE TITLE
Slide printing not adding page breaks in the right spots

### ISSUE THREAD

**Issue by IronhandedLayman** [webpro/reveal-md#97]
When I attempt to print slides from reveal-md (0.1.2) using PhantomJS (version 2.1.1), the slides will go into a .pdf, but the page breaks are all wrong (slides are being broken by the page breaks).

**Comment by webpro** [comment]
Yes, we're waiting on a reveal.js release for this (it's fixed in reveal's master branch).

**Comment by zerkalica** [comment]
Still not fixed, may be some workaround?

**Comment by webpro** [comment]
Maybe you can ask the reveal.js maintainer(s) to release the latest changes, that should fix this issue.

**Comment by zerkalica** [comment]
Ok, but i don't know which pr link. Can you link this issue with suitable reveal.js pr?

**Comment by webpro** [comment]
Last time I looked, it was already in their master branch. Just not released yet.

**Comment by zerkalica** [comment]
I found some issues in reveal.js, but don't sure that they related to this problem:

https://github.com/hakimel/reveal.js/issues/1648
https://github.com/hakimel/reveal.js/issues/1804

**Comment by webpro** [comment]
Please try the following (psuedo):

```
git clone reveal-md
cd reveal-md
npm install git://reveal.js#master
npm link
```

Then try `reveal-md slides.md` on your slides.

**Comment by deeuu** [comment]
Hi,

Printing is also no good for me. I tried the above but still no luck.

Note that print [example instructions](https://github.com/hakimel/reveal.js/#instructions-1) work fine.

Would you mind explaining the problem to me, please?

Thanks

**Comment by webpro** [comment]
The required fix of Reveal.js has been released, just after my last comment. I've updated the dependency and this is released in [v0.3.0](https://github.com/webpro/reveal-md/releases/tag/0.3.0). Please let me know if PDF output is better now.

**Comment by zerkalica** [comment]
Yes, bug fixed for me. Page breaks in pdf works fine.
